lib/bulk/ordered.js

```diff
@@ -515,6 +515,17 @@ var executeCommands = function(self, options, callback) {
     finalOptions.checkKeys = false;
   }
 
+  if (finalOptions.retryWrites) {
+    if (batch.batchType === common.UPDATE) {
```

Comment:
why not for insert?

Reply:
`insertMany` is supported, so we don't need to disable the bulkWrite for it.